index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Nova</title>
+
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/base.css">
+    <link rel="stylesheet" href="./styles/header.css">
+    <link rel="stylesheet" href="./styles/description.css">
+    <link rel="stylesheet" href="./styles/features.css">
+    <link rel="stylesheet" href="./styles/footer.css">
+    <link rel="stylesheet" href="./styles/pricing.css">
+    <link rel="stylesheet" href="./styles/sponsors.css">
+    <link rel="stylesheet" href="./styles/team.css">
+</head>
+<body>
+    <header class="header row">
+        <div class="nav__container container">
+            <img src="./images/logo.svg" alt="Nova logo" class="nav__logo cell">
+            <nav class="nav__menu cell">
+                <ul class="menu">
+                    <li class="menu__item"><a href="#team-section">The Team</a></li>
+                    <li class="menu__item"><a href="#pricing-section">Pricing</a></li>
+                    <li class="menu__item"><a href="#features-section">Features</a></li>
+                </ul>
+            </nav>
+        </div>
+        
+        <div class="header__container container">
+            <article class="header__article cell">
```

Comment:
Pokazywanie scrolla nie jest dobrym pomysłem:
![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/7156e2d0-1037-4721-a3af-49810b5b3eff)


W następnym materiale będzie jak robić bardziej elastyczne (RWD) elementy.